css/style.css

```diff
@@ -689,6 +699,42 @@ ul.chars{
     margin: 0 auto;
 }
 
+@media only screen and (max-device-width:1742px) {
+    .page-wide {
+        width: 100%;
+        margin: 0 auto;
+    }
+    a.nav {
+        text-align: center;
+        font-size:0.5em
+    }
+    .nav-pages {
+        float: none;
+    }
+    .brand-logo {
+        display: none;
+    }
+    .brand-link {
+        float: none;
+    }
+    .home-text {
+        display: block !important;
```

Comment:
you could move the other home text selector above this block and not need important here